sdk/core/core-http/lib/policies/logPolicy.ts

```diff
@@ -60,6 +76,17 @@ const defaultAllowedQueryParameters: string[] = [
   "api-version"
 ];
 
+export const DefaultLogPolicyOptions: LogPolicyOptions = {
```

Comment:
The `DefaultLogPolicyOptions` isnt used anywhere

Reply:
Damn, I swear I deleted that earlier.  Gone now